File: go/run demo/Spot Trade/CancelOrder.go
package main

import (
	"demo/spot"
	"fmt"
)

//说明：
//在params中输入json格式的参数,如:`{"symbol":"BTCUSDT",	"limit":"200"}`
//如果没有参数则输入:""

//现货参数
// var params string = ""
var params string = `{}`

func main() {

	//接口调用
	CancelOrder := spotList.CancelOrder(params)
	fmt.Println("返回信息:", CancelOrder)

}

File: python/run demo/Spot Trade/CancelOrder.py
from python.spot import mexc_spot_v3


trade = mexc_spot_v3.mexc_trade()

# Enter parameters in JSON format in the "params", for example: {"symbol":"BTCUSDT", "limit":"200"}
# If there are no parameters, no need to send params
params = {
    "symbol": "MXUSDT",
    "orderId": "C01__xxx"
}
CancelOrder = trade.delete_order(params)
print(CancelOrder)